scripts/publish.sh

```diff
@@ -118,6 +118,20 @@ publish_to_docker_hub() {
 
 }
 
+publish_to_public_ecr() {
+	aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws/aws-observability
```

Comment:
I am not sure why its default to `us-east-1`? Is it for public ECE?

Reply:
Yes, for public ECR, we always set the region to `us-east-1`

Reply:
Maybe I don't have enough insights here. For public ECR is it necessary to hit a regional endpoint for getting login-password?

Reply:
Yes we should: https://docs.aws.amazon.com/AmazonECR/latest/public/getting-started-cli.html. We still need to authenticate to the public registry but specify `--region us-east-1`.